docker/validator/setup.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/sh
 
+set -ue
+
 git clone https://github.com/CosmWasm/wasmd.git
 cd wasmd
 git checkout "${WASMD_VERSION}"
+WASMD_COMMIT_HASH=$(git rev-parse HEAD)
```

Comment:
We could but then it'd be possible to select a commit hash that doesn't correspond to the version. Is that really a desirable feature? If someone wants to build a specific commit right now, they can specify `WASMD_VERSION` as a commit hash instead and it will build that commit.

Since tags have a 1:1 correspondence to commit hashes, it seems like you'd want to take one or the other, but not both.